dags/web_preaggregated.py

```diff
@@ -0,0 +1,220 @@
+from datetime import datetime, timedelta
+import os
+from typing import Optional
+from collections.abc import Callable
+
+import dagster
+from dagster import Field, Array, Definitions
+
+from clickhouse_driver import Client
+from dags.common import ClickhouseClusterResource
+from posthog.models.web_preaggregated.sql import (
+    WEB_OVERVIEW_METRICS_DAILY_SQL,
+    DISTRIBUTED_WEB_OVERVIEW_METRICS_DAILY_SQL,
+    WEB_STATS_DAILY_SQL,
+    DISTRIBUTED_WEB_STATS_DAILY_SQL,
+    WEB_OVERVIEW_INSERT_SQL,
+    WEB_STATS_INSERT_SQL,
+)
+from posthog.clickhouse.cluster import ClickhouseCluster
+
+WEB_ANALYTICS_DAILY_PARTITION_DEFINITION = dagster.WeeklyPartitionsDefinition(
+    start_date="2025-01-01",
+    fmt="%Y-%m-%d",
+    timezone="UTC",
+)
+
+DEFAULT_TEAM_IDS = [int(id) for id in os.getenv("WEB_ANALYTICS_TEAM_IDS", "").split(",") if id]
```

Comment:
(I know this a draft PR) I'd call the env var something more descriptive than `WEB_ANALYTICS_TEAM_IDS`, otherwise someone seeing it in config somewhere with no config might not be clear what it does

Reply:
Good comment! I've removed the env config; it was only to test it locally. We can provide a custom one when launching the Dagster jobs on their "launchpad" and even save that config for some manual runs.